crl/updater/updater.go

```diff
@@ -62,6 +62,10 @@ func NewUpdater(
 		return nil, fmt.Errorf("must update CRLs at least every 7 days, got: %s", updatePeriod)
 	}
 
+	if updateOffset > updatePeriod {
+		updateOffset = updatePeriod
```

Comment:
Isn't this the same as setting updateOffset = 0?

Clamping seems like a surprising behavior here. I would expect to error in this case.

Reply:
Turned this into an error and updated the config comment.